server/interpreter.js

```diff
@@ -1837,6 +1835,82 @@ Interpreter.prototype.pushNode_ = function(node) {
 //   small performance improvement in V8.
 
 /**
+ * @constructor
+ * @param {number} id
+ * @param {string|!Object} code
```

Comment:
I now have some JSDoc output to look at, but it makes me think we should delay worrying about this for now.